index.php

```diff
@@ -220,7 +228,7 @@ function validateTicket($file) {
 	// A file is valid if it has an allowed extension,
 	// an allowed filetype and is less than or equal
 	// in size to the maximum filesize.
-	if($file['size'] <= FILE_MAX_FILESIZE ){
+	if($file['size'][$index] <= FILE_MAX_FILESIZE ){
```

Comment:
I suppose this works, but wouldn't `$file[$index]['size']` be a lot more readable? I guess that would mean a bit of refactor

Reply:
Agreed!
It also got me itching when porting these changes over from the fork, and rewriting files to be AoS is a nice to have for readability if the implementation is as simple.

So I will look into this rewrite soon, will draft this PR for now :D

Reply:
I'm looking forward to the next review :)